northstar/src/runtime/island/fs.rs

```diff
@@ -39,23 +39,23 @@ pub(super) struct Mount {
     pub fstype: Option<&'static str>,
     pub flags: MsFlags,
     pub data: Option<String>,
+    pub err_str: String,
 }
 
+// TODO: The container could be malformed and the mountpoint might be missing.
+// This is not a fault of the RT so don't expect it.
 impl Mount {
     /// Execute this mount call
-    pub(super) fn mount(&self) {
-        if !self.target.exists() {
-            panic!("Missing mount point {}", self.target.display())
-        }
-
+    pub(super) fn mount(&self) -> Result<(), ()> {
```

Comment:
The `Result<(), ()>` doesn't make sense and is unused. There's the `expect` which is fine.